libnext/src/models/types/kinds.ts

```diff
@@ -0,0 +1,16 @@
+export enum Kind {
+  Null = "null",
```

Comment:
Styling note: personally, I prefer enum cases to be `UPPERCASE` as it makes it easier to separate them from types. Up to you though.